README.md

```diff
@@ -1,2 +1,44 @@
-# HTML-CSS-JavaScript-capstone-project
-This capstone project I built is for an online website for a leadership training. I have used HTML, CSS and JavaScript to build this website.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> In this first capstone project, I have built an online website for a leadership conference..
+
+
+## Built With
+
+- HTML 
+- CSS
+- JavaScript
+
+## Live Demo (if available)
+
+[Live Demo Link](http://rn486.github.io/Capstone1-project/)
```

Comment:
- [ ] Please your live demo link has a broken CSS style link so it display only the HTML structure page
### [live demo](https://rn486.github.io/Capstone1-project/)
![live](https://user-images.githubusercontent.com/55596970/181654803-ab984a83-77a3-4a27-a1a5-37e859606bdb.png)

Reply:
Thank you for the comments, but I would like pointers on the deployment part. I have not found a solution to make it visible.

Reply:
Ok, 
Please can we have a call to check it out?

Reply:
Yes, please. Thank you.

Reply:
Join Zoom Meeting
https://microverse.zoom.us/j/93366219619?pwd=eXRiNWIwRWl3QjRSTHdBTTVwVXZvUT09

Meeting ID: 933 6621 9619
Passcode: 925314
One tap mobile
+13017158592,,93366219619#,,,,*925314# US (Washington DC)
+13126266799,,93366219619#,,,,*925314# US (Chicago)

Dial by your location
        +1 301 715 8592 US (Washington DC)
        +1 312 626 6799 US (Chicago)
        +1 346 248 7799 US (Houston)
        +1 386 347 5053 US
        +1 564 217 2000 US
        +1 646 931 3860 US
        +1 669 444 9171 US
        +1 669 900 6833 US (San Jose)
        +1 929 205 6099 US (New York)
        +1 253 215 8782 US (Tacoma)
Meeting ID: 933 6621 9619
Passcode: 925314
Find your local number: https://microverse.zoom.us/u/adgv23LQHT